### PR TITLE
lazy load article images

### LINKED ISSUE #294: Lazy load all iframes with a size
Native lazy loading is coming: https://caniuse.com/#feat=loading-lazy-attr 
It's been [in Chrome](https://www.chromestatus.com/feature/5645767347798016) since v75-76 (as of this writing, 79 is release). 
It's actually very easy to implement. You just put `loading="lazy"` on your `iframe` or `img` tags but you need to have the size dimensions set, which we have for our iframes. 

All it takes is 15 characters and if it works it helps, if it doesn't work it's fine. 

It certainly might be nice to do a full IntersectionObserver thing on it but those things can be tricky because you have to not make it worse for really old browsers and things like [lazysizes](https://github.com/aFarkas/lazysizes) is 7KB even though it's already pretty compact.

### LINKED ISSUE #1540: Is there any reason to not lazy load images?
In Yari we control every `` tag and it would be trivial to inject a `loading="lazy"` attribute. 
Lazy loading images have the advantage that there's less to do on initial load so the main thread has less to worry about. 
And if you're on a fast desktop computer with a fast connection, it might be pleasant to not have to see the images flicker in as you scroll down. 

Lots of background and tips here: https://web.dev/browser-level-image-lazy-loading/
One interesting point they're making is that it's not great to use lazy loading on images *above* the fold. Something that might be hard to figure out and understand in the Yari builder. 

But without a doubt, many people do not scroll down and if there's even a small chance that they are on a weaker connection, we'd be doing them a service for setting `loading="lazy"`. 

Let this issue be about the research into **if** rather than just doing it.

### PR DESCRIPTION
**Pull request by peterbe**
Fixes #1540
Fixes #294

@Keith-CY can you test this? It seems to work but perhaps you can test it more carefully on more pages.

### CODE REVIEW

**Comment by Keith-CY** [comment]
> Fixes #1540
> 
> @Keith-CY can you test this? It seems to work but perhaps you can test it more carefully on more pages.

Glad to do it :)

**Comment by Keith-CY** [comment]
I've checked several pages and they're all working as expected.

**Review by Keith-CY — APPROVED** [review]

File: build/index.js [hunk 1]
```diff
@@ -93,6 +93,15 @@ function injectNoTranslate($) {
   $("pre").addClass("notranslate");
 }
 
+/**
+ * For every image and iframe, where appropriate add the `loading="lazy"` attribute.
```
Comment (Keith-CY) [inline build/index.js:97]: `iframe` is not updated in this commit, better to fix the comment.

Comment (peterbe) [reply build/index.js:97]: Heck, I might as well go for it.

**Comment by peterbe** [comment]
@Keith-CY I've made it work for iframes too. It still only works in Chrome but it definitely works there. I quite like it. 
There are a lot of iframes that don't appear until late in the page and it's totally realistic that you never scroll down enough to actually load it. I use Firefox but a lot of people use Chrome, so they'll benefit from this. 

Can you test it some more please?

**Comment by Keith-CY** [comment]
> @Keith-CY I've made it work for iframes too. It still only works in Chrome but it definitely works there. I quite like it.
> There are a lot of iframes that don't appear until late in the page and it's totally realistic that you never scroll down enough to actually load it. I use Firefox but a lot of people use Chrome, so they'll benefit from this.
> 
> Can you test it some more please?

Cool, it's been confirmed that working with `iframe` in Chrome.

**Pull request merged**